go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/tailscale/tailscale-synology
 
 go 1.17
 
-require github.com/ulikunitz/xz v0.5.10 // indirect
+require (
+	github.com/ulikunitz/xz v0.5.10 // indirect
+	tailscale.com v1.1.1-0.20230211072955-b64d900f0ffb // indirect
```

Comment:
what's with this addition?

Reply:
Looks like a side effect of me running `go get tailscale.com@HEAD`, assuming that tailscale.com was used by this repo somehow. Turns out nope, it's not used in this repo, so a `go mod tidy` reverted these changes. Fixed.